docs/WS_ORDER_EVENTS.md

```diff
@@ -0,0 +1,31 @@
+# Websocket trade events
+
+GoCryptoTrader unifies trades and order update events by composing an
+order.Detail object.  This is the full list of order.Detail fields
+that exchange implementations should populate on streamed
+trade/order-update events.  As exchanges provide different APIs, not
+all fields are mandatory.
+
+```
+| order.Detail field   | Description                                                       | Condition                                               | Presence  |
```

Comment:
One pedantic nit: We could drop the code block and have it as a table instead so it can resize a bit better.

Like here: 
![image](https://user-images.githubusercontent.com/9391715/129982939-50ac76b3-6be3-4945-a0b6-d56f12af169a.png)

Instead of: 
![image](https://user-images.githubusercontent.com/9391715/129982863-f6dbe671-0410-4275-998a-988458660b90.png)